templates/empty/index.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * Generated using theia-plugin-generator
```

Comment:
I was thinking, when reviewing this sentence I wrote that we should better say

Generated by @theia/generator-plugin yeoman generator using `yo @theia/plugin` command or something like that so it's easy to reproduce it when looking at the code

Reply:
WDYT for the help on top of the files ?